pyproject.toml

```
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["hatchling>=1.4.0", "jupyterlab>=3.1.0,<5.0.0", "hatch-nodejs-version"]
+requires = ["hatchling>=1.4.0", "jupyterlab>=3.4.7,<4.0.0", "hatch-nodejs-version"]
```

This change should be undone
```suggestion
requires = ["hatchling>=1.4.0", "jupyterlab>=3.1.0,<5.0.0", "hatch-nodejs-version"]
```